content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,33 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based exclusion filter
```

Comment:
We are already in the exclusion filter section so we could rename this section `Attribute based Sampling` ?